src/components/app/details/appDetails/DeploymentStatusBreakdown.tsx

```diff
@@ -51,19 +57,23 @@ export default function DeploymentStatusDetailBreakdown({
         )
     }
     return (
-        <div className="deployment-status-breakdown-container pl-20 pr-20">
-            {deploymentStatusDetailsBreakdownData.deploymentError && (
-                <InfoColourBar
-                    message={deploymentStatusDetailsBreakdownData.deploymentError}
-                    classname="error_bar cn-9 mb-20 lh-20"
-                    Icon={Error}
-                    iconClass="icon-dim-20"
-                />
-            )}
-            {renderStatusDetailRow('DEPLOYMENT_INITIATED')}
-            {renderStatusDetailRow('GIT_COMMIT')}
-            {renderStatusDetailRow('KUBECTL_APPLY')}
-            {renderStatusDetailRow('APP_HEALTH', true)}
-        </div>
+        <>
+            {url.includes(`/${URLS.APP_CD_DETAILS}`) && <ErrorBar appDetails={_appDetails} />}
```

Comment:
Should it be {!url.includes(`/${URLS.APP_CD_DETAILS}`) && <ErrorBar appDetails={_appDetails} />}? As we will be hiding the error baar on cd details page